regulations/static/regulations/js/source/views/sidebar/sidebar-view.js

```diff
@@ -75,7 +75,7 @@ var SidebarView = Backbone.View.extend({
     },
 
     updateChildViews: function(context) {
-        var $definition = $definition || this.$el.find('#definition');
+        var $definition = $definition || this.$el.find('#definition'); // eslint-disable-line no-use-before-define
```

Comment:
Er, now that you mention it, I don't see `$definition` defined elsewhere in the source. I'm guessing this is meant to lazily cache `this.$el.find('#definition')` in case `updateChildViews` is called repeatedly. I'll replace `$definition` with `this.$definition`.